src/classes/create-session.ts

```diff
@@ -3,8 +3,8 @@ import { TCreateOptions } from '../types/create-options';
 import { ISequelizeService } from '../interfaces/sequelize-service';
 import { ICreateSession } from '../interfaces/create-session';
 
-export class CreateSession<W, R extends W, C> extends Session<W, R, C, TCreateOptions<R, C>> implements ICreateSession<W, R, C> {
-  public constructor(objects: W[], options: TCreateOptions<R, C>, service: ISequelizeService<W, R, C>) {
+export class CreateSession<W extends {}, R extends W, C extends {}, Compute extends keyof C> extends Session<W, R, C, TCreateOptions<R, C, Compute>> implements ICreateSession<W, R, C, Compute> {
```

Comment:
`Compute` should be shortened

Reply:
Ideas?

`RS` and `CS` ? 🤔 

Google's Java multi-letter generic convention is to put a `T` in the end, i.e. `ComputeT` and `SelectT`.

Reply:
RS and CS will do. As per having a better / more descriptive convention, I'm open to it but then all generics should follow it.

Reply:
Let's do `KR` and `KC` (read: `keys of R` and `keys of C`)